### ISSUE TITLE
[feature request] flexible heredoc / nowdoc

### ISSUE THREAD

**Issue by mathroc** [squizlabs/PHP_CodeSniffer#2317]
I'd like to be able to force usage of flexible heredoc and nowdoc (php 7.3)

eg: those should trigger an error when the sniff is enabled:

```php
    a_function(<<<EOS
a string
EOS); // EOS should be indented
```

```php
    a_function(<<<EOS
    a string
    EOS
); // should be on the same line as the ending EOS token
```

```php
    a_function(<<<EOS
    a string
    EOS); // correct !
```
```php
    a_function(<<<EOS
        a string
    EOS); // correct !
```
